app/scripts/controllers/transactions/bitbox-utils.js

```diff
@@ -155,14 +155,29 @@ class BitboxUtils {
           )
         }
 
+        const isCashAccountRegistration =
+          txParams.isCashAccountRegistration !== undefined &&
+          txParams.isCashAccountRegistration
+
+        if (isCashAccountRegistration) {
```

Comment:
Is it possible to use the opReturn.position property for this, checking for position 0 instead of `isCashAccountRegistration` and position 1 instead of ! isCashAccountRegistration ?

Reply:
yes, but I didn't know what the default position should be, and all the potential places it might affect.

This way I know only cashaccount registrations will have any position logic.